### ISSUE TITLE
Unable to disable drawEmpty for pie series

### ISSUE THREAD

**Issue by eshpatil** [highcharts/highcharts#12663]
#### Expected behaviour

word map with pie chart should render correctly

#### Actual behaviour

chart is not rendering correctly
![world-map-with-pie](https://user-images.githubusercontent.com/6734528/71154305-3683c980-2261-11ea-9b99-58ba5e46be54.PNG)

#### Live demo with steps to reproduce
http://jsfiddle.net/tbf57ejc

#### Product version
8.0.0

#### Affected browser(s)
All

**Comment by sebastianbochan** [comment]
Hi @eshpatil,
Thank you for reporting about the problem. It looks like you commented some parts of code and that is the reason why its not working. Could you elaborate the reason of that?

When I open the demo: https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/maps/demo/map-pies/ all worked properly.

**Comment by eshpatil** [comment]
Hi,
Yes its working fine if we render map with pie for US region but when we try to render it for full world map it is not rendering correctly.
If I used older version of highchart (7.2.1) with same code its working correctly for full world map, but when I user latest version (8.0.0) it is not working.

**Comment by eshpatil** [comment]
Its working fine if we draw pie with US map region but when we try to draw pie with full world map it it not rendering properly, and same is working correctly till version 7.1.3

Following is working demo with version 7.1.3
https://jsfiddle.net/xf2goyrj/

And exactly same code with latest version is not working
https://jsfiddle.net/o5Ld0yrb/

**Comment by KacperMadej** [comment]
As a workaround you could disable the empty pies by adding the below code:

```js
// don't draw empty pies
Highcharts.seriesTypes.pie.prototype.drawEmpty = function(){};
```
Demo: https://jsfiddle.net/BlackLabel/q27nxpv3/

**Comment by KacperMadej** [comment]
Another workaround could be to set `borderWidth` for the pie type series to `0`.
The elements will be rendered, but `stroke-width` of those elements will be `0`, so they won't be visible.
Demo: https://jsfiddle.net/BlackLabel/q27nxpv3/1/

**Comment by madepiet** [comment]
Related to: #13229

**Comment by madepiet** [comment]
Closed via: #13543
